setup.py

```diff
@@ -19,8 +19,8 @@
 
 name = "dialogflow"
 description = "Client library for the Dialogflow API"
-version = "0.8.0"
-release_status = "Development Status :: 4 - Beta"
+version = "1.0.0"
```

Comment:
Would you mind just reverting this line? I think release-please will try to bump it again.

Reply:
Done!